packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -216,6 +216,35 @@ void main() {
     expect(transformWidget.transform.getRotation()[4], 1.0);
   });
 
+  testWidgets('UserAccountsDrawerHeader icon color changes', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Material(
+        child: UserAccountsDrawerHeader(
+              onDetailsPressed: () { },
+              accountName: const Text('name'),
+              accountEmail: const Text('email'),
+        ),
+      ),
+    ));
+
+    Icon iconWidget = tester.firstWidget(find.byType(Icon));
+    expect(iconWidget.color, Colors.white);
+
+    await tester.pumpWidget(MaterialApp(
+      home: Material(
+        child: UserAccountsDrawerHeader(
+          onDetailsPressed: () { },
+          accountName: const Text('name'),
+          accountEmail: const Text('email'),
+          arrowColor: const Color.fromRGBO(255, 0, 0, 1.0),
+        ),
+      ),
+    ));
+
+    iconWidget = tester.firstWidget(find.byType(Icon));
+    expect(iconWidget.color, const Color.fromRGBO(255, 0, 0, 1.0));
```

Comment:
Maybe consider declaring a variable for this color and using it here and on line 239.  Not a big deal, but I'm just thinking it would make it more obvious.  Up to you.

Reply:
Agree with this as well. Declare a variable or even use an existing Color like `Colors.green`.

Reply:
Actually I was going to use `Colors.red`, but it can't be used, as it's a `MaterialColor` and not a `Color`. I agree it looks way better as a variable, easier to change and read in the future if necessary.

Reply:
Funny thing, I remember IntelliJ complaining about `Colors.red` being a `MaterialColor` and couldn't be used as a `Color`, but now it works, go figure. I think my IntelliJ isn't much happy with the Flutter repo, I am getting some behaviors that I don't get with my apps.

Reply:
I think I got it, it complains about `arrowColor: const Colors.red,`, but doesn't for `arrowColor: arrowColor,`, where `arrowColor` is `const Color arrowColor = Colors.red;`, my Dart Fu isn't strong enough to understand why, as to me they are all `const Color`.

Reply:
I thought both would be fine too :man_shrugging: